exercises/swift-scheduling/canonical-data.json

```diff
@@ -0,0 +1,133 @@
+      "uuid": "25f46bf9-6d2a-4e95-8edd-f62dd6bc8a6e",
+      "description": "EOW on Monday translates to Friday at five in the afternoon",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2025-02-03T00:00:00",
+        "description": "EOW"
+      },
+      "expected": "2025-02-07T17:00:00"
+    },
+    {
+      "uuid": "0b375df5-d198-489e-acee-fd538a768616",
+      "description": "EOW on Tuesday translates to Friday at five in the afternoon",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "1997-04-29T00:00:00",
+        "description": "EOW"
+      },
+      "expected": "1997-05-02T17:00:00"
+    },
+    {
+      "uuid": "4afbb881-0b5c-46be-94e1-992cdc2a8ca4",
+      "description": "EOW on Wednesday translates to Friday at five in the afternoon",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2005-09-14T00:00:00",
+        "description": "EOW"
+      },
+      "expected": "2005-09-16T17:00:00"
+    },
+    {
+      "uuid": "e1341c2b-5e1b-4702-a95c-a01e8e96e510",
+      "description": "EOW on Thursday translates to Sunday at eight in the evening",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2011-05-19T00:00:00",
+        "description": "EOW"
+      },
+      "expected": "2011-05-22T20:00:00"
+    },
+    {
+      "uuid": "bbffccf7-97f7-4244-888d-bdd64348fa2e",
+      "description": "EOW on Friday translates to Sunday at eight in the evening",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2022-08-05T00:00:00",
+        "description": "EOW"
+      },
+      "expected": "2022-08-07T20:00:00"
+    },
+    {
+      "uuid": "439bf09f-3a0e-44e7-bad5-b7b6d0c4505a",
+      "description": "2M translates to the first workday of the second month",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2007-01-02T00:00:00",
+        "description": "2M"
+      },
+      "expected": "2007-02-01T08:00:00"
+    },
+    {
+      "uuid": "86d82e83-c481-4fb4-9264-625de7521340",
+      "description": "5M translates to the first workday of the fifth month",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2013-02-11T00:00:00",
+        "description": "5M"
+      },
+      "expected": "2013-05-01T08:00:00"
+    },
+    {
+      "uuid": "06d401e3-8461-438f-afae-8d26aa0289e0",
+      "description": "Q1 translates to the last workday of the first quarter",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2011-02-23T00:00:00",
+        "description": "Q1"
+      },
+      "expected": "2011-03-31T08:00:00"
+    },
+    {
+      "uuid": "eebd5f32-b16d-4ecd-91a0-584b0364b7ed",
+      "description": "Q4 translates to the last workday of the fourth quarter",
+      "property": "deliveryDate",
+      "input": {
+        "meetingStart": "2023-04-19T00:00:00",
+        "description": "Q4"
+      },
+      "expected": "2023-12-29T08:00:00"
```

Comment:
How about a test case that wraps around to the next year?

Reply:
I like that idea. I think it makes the most sense to have that as part of the ASAP tests. I'll add it

Reply:
On second hand, the quarters suggestion is good! I've updated the description, I'll add the test cases later. Could you check if the description makes sense?

Reply:
I've added test cases too